docs/guide/dev/deep_dive/errors/README.md

```diff
@@ -23,6 +23,18 @@ This article contains the list of CKEditor error codes and their explanation. Th
 * Description: Incorrect {@linkapi CKEDITOR.config#autoEmbed_widget } value. No widget definition found.
 * Additional data: None.
 
+## cloudservices-no-url
+
+* Location: `plugins/cloudservices/plugin.js`
+* Description: Upload URL used by {@linkapi CKEDITOR.plugins.cloudServices.cloudServicesLoader} is not set. An URL can be provided by configuration {@linkapi CKEDITOR.config#url} property or passed to [Cloud Services](https://ckeditor.com/cke4/addon/cloudservices) plugin {@linkapi CKEDITOR.plugins.cloudServices.cloudServicesLoader#upload} method.
```

Comment:
Sentence:

> Upload URL used by CKEDITOR.plugins.cloudServices.cloudServicesLoader is not set.

Is incorrect. Please read the whole change proposed in PR to make sure there are no more issues like that. You want to generally read (sometimes even aloud) things you have written, before pushing your commit.

Reply:
You're right with pointing out that the URL could be customized in `cloudServicesLoader#upload` method, but I feel we're diving a bit too deep here. Let's stick to a more generic relation - I mean, you should talk about Cloud Services plugin and not particular API function names.

So you want to remove part about possible URL customization (this should be documented in API docs of a proper method/argument) and you don't want to relate to a particular API function.